registry/time/functions_test.go

```diff
@@ -2,39 +2,115 @@ package time_test
 
 import (
 	"testing"
-	goTime "time"
+	"time"
```

Comment:
If you decide to reverse the logic like that can you check we don't have another "std import with alias" on the codebase to be consistant please